### ISSUE TITLE
Youtube custom element isn't working correctly with UI elements

### ISSUE THREAD

**Issue by heff** [muxinc/media-chrome#87]
In the following example the controls don't seem to be interacting with the media element, e.g. the play button starts with the "pause" icon showing.

https://media-chrome-err0093s8-mux.vercel.app/examples/media-elements/youtube.html

This is probably related to https://github.com/muxinc/youtube-video-element/pull/9

**Comment by cjpillsbury** [comment]
Yeah the current implementation of `<youtube-video-element/>` causes RTEs on initialization because of the assumed functioning API on the element when it's slotted to `media` in `<media-controller/>`.
